### PR TITLE
Add warning for deprecated Glances version

### PR DESCRIPTION
**Pull request by engrbm87**
## Proposed change

Add warning for deprecated Glances version

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/105887
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/glances.markdown [hunk 1]
```diff
@@ -39,6 +39,12 @@ If this doesn't work, try changing the `3` to `2`, if you don't have the latest
 
 For details about auto-starting `glances`, please refer to [Start Glances through Systemd](https://github.com/nicolargo/glances/wiki/Start-Glances-through-Systemd).  
 
+<div class='note warning'>
```
Comment (MartinHjelmare) [inline source/_integrations/glances.markdown]: I think we should move the warning to right under the "Setup" header.

Also, I think we should consider removing the text about testing the API and using different versions. Users should just start the service and then try to set up the integration in Home Assistant. We'll now try both version 3 and 2 automatically.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/glances.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_integrations/glances.markdown]: @engrbm87, are you good with the changes to go into this beta?

Comment (engrbm87) [reply source/_integrations/glances.markdown]: I believe Martin had some suggestions, but unfortunately I don't have time at the moment to address them.
If you can include these suggestions then we should be good to go.

Comment (c0ffeeca7) [reply source/_integrations/glances.markdown]: ok, will do. Thank you for the feedback!

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Review by frenck — APPROVED** [review]

**Pull request merged**